build-logic/build-platform/build.gradle.kts

```diff
@@ -14,7 +14,8 @@ dependencies {
     constraints {
         // Gradle Plugins
         api("com.gradle:gradle-enterprise-gradle-plugin:3.5.2")
-        api("com.gradle.enterprise:test-distribution-gradle-plugin:1.3.1")
+        // Keep version with `settings.gradle.kts` in sync
+        api("com.gradle.enterprise:test-distribution-gradle-plugin:2.0-rc-3")
```

Comment:
We will collect all settings plugin in a convention plugin for settings. Then we won't have to duplicate the version anymore. We can keep it like is is here. We'll have to make the change when we rebase #15526.

Reply:
👍